pkg/kubelet/cm/topologymanager/scope_pod.go

```diff
@@ -62,6 +62,10 @@ func (s *podScope) Admit(pod *v1.Pod) lifecycle.PodAdmitResult {
 			return admission.GetPodAdmitResult(err)
 		}
 	}
+	if IsAlignmentGuaranteed(s.policy) {
+		// increment only if we know we allocate aligned resources.
+		metrics.ContainerAlignedComputeResources.WithLabelValues(metrics.AlignedNUMAZone).Inc()
```

Comment:
should we separate pod and container alignment? do we expect a container to be aligned IFF a pod is aligned? if so, will the value of 2 per aligned container be confusing?

Reply:
The semantic can be confusing but I'm struggling to make it easier to consume if one's not deep into details (which will likely bring in turn questions about the design of the features, but let's not digress now).

with scope=pod, the topology manager (TM) will place ALL the pod's containers on the same NUMA zone, failing to admit if not possible. So if we are here, it means all the containers are BOTH aligned AND all on the _same_ NUMA zone.

with scope=container, TM will place ALL the pod's containers on ANY available NUMA zone, failing to admit if any container can't get aligned resources. So, if we are here, it means ALL containers got NUMA aligned resources, but they can be scattered on different NUMA zones.

Reply:
so I guess the question is: should the metric footprint look different for those two cases?  would an admin want to quicky tell what it meant by alignedNUMAZone? if they have distinct code paths and distinct results, it seems like something that would warrant an  additional label value

Reply:
good point. I can add a metric label "scope" whose value can be "pod" or "container"